lightningd/gossip_msg.c

```diff
@@ -58,8 +60,10 @@ void towire_gossip_getchannels_entry(
 	towire_pubkey(pptr, &entry->source);
 	towire_pubkey(pptr, &entry->destination);
 	towire_bool(pptr, entry->active);
-	towire_u32(pptr, entry->fee_per_kw);
-	towire_u32(pptr, entry->delay);
-	towire_u32(pptr, entry->last_update_timestamp);
 	towire_u16(pptr, entry->flags);
+	towire_u64(pptr, entry->last_update_timestamp);
```

Comment:
Hm, I kind of dislike the idea of pushing this logic down to the serialization layer, but the only alternative I can come up with is passing dummy values like we did before, so I'm ok with doing it this way.